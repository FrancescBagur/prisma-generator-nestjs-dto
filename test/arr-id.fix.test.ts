import { describe, test } from 'vitest';

import { testFixtures } from './fixtures';

describe('fixtures', () => {
  const BASIC = 'fixtures/arr-id.prisma';
  test(BASIC, testFixtures(BASIC));
});
