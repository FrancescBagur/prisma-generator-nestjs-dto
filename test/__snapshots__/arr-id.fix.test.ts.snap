// Vitest Snapshot v1

exports[`fixtures > fixtures/arr-id.prisma 1`] = `
[
  {
    "content": "
  export class ConnectProductDto {
  id: string;
  anotherId: string;
  }
  ",
    "fileName": "../src/@generated/prisma-generator-nestjs-dto/connect-product.dto.ts",
  },
  {
    "content": "
import {Prisma} from '@prisma/client'


export class CreateProductDto {
  anotherId: string;
  name: string;
  description: string;
  images: Prisma.InputJsonValue[];
  highlighted?: boolean;
  reviewCount?: number;
  attributes?: Prisma.InputJsonValue;
  score?: number;
  decimal?: Prisma.Decimal;
  categoryId: string;
  companyId: string;
}
",
    "fileName": "../src/@generated/prisma-generator-nestjs-dto/create-product.dto.ts",
  },
  {
    "content": "
import {Prisma} from '@prisma/client'


export class UpdateProductDto {
  id: string ;
  anotherId: string ;
  name?: string ;
  description?: string ;
  images?: Prisma.JsonValue[] ;
  highlighted?: boolean  | null;
  reviewCount?: number  | null;
  attributes?: Prisma.JsonValue  | null;
  score?: number  | null;
  decimal?: Prisma.Decimal  | null;
  categoryId?: string ;
  companyId?: string ;
}
",
    "fileName": "../src/@generated/prisma-generator-nestjs-dto/update-product.dto.ts",
  },
  {
    "content": "
import {Prisma} from '@prisma/client'
import {IntersectionType} from '@nestjs/swagger'

export class Product {
  id: string ;
  anotherId: string ;
  name: string ;
  description: string ;
  images: Prisma.JsonValue[] ;
  highlighted: boolean  | null;
  reviewCount: number  | null;
  attributes: Prisma.JsonValue  | null;
  score: number  | null;
  decimal: Prisma.Decimal  | null;
  categoryId: string ;
  companyId: string ;
}
export class ProductRel {
  
}
export class ProductFull extends IntersectionType(Product, ProductRel) {}
",
    "fileName": "../src/@generated/prisma-generator-nestjs-dto/product.entity.ts",
  },
]
`;
